subsys/bluetooth/host/hci_core.c

```diff
@@ -111,8 +111,12 @@ struct cmd_data {
 
 static struct cmd_data cmd_data[CONFIG_BT_BUF_CMD_TX_COUNT];
 
+#if defined(CONFIG_BT_CONN)
+static struct acl_data acl_data[CONFIG_BT_BUF_ACL_RX_COUNT];
```

Comment:
This needs  a fix in `conn_internal.h`:

```
struct acl_data {
	/* Extend the bt_buf user data */
	struct bt_buf_data buf_data;
```

Reply:
also moved the struct def since it's only used in hci_core.c now